prime-router/metadata/hl7_mapping/ORU_R01/base/telecom.yml

```diff
@@ -0,0 +1,6 @@
+name: ORU-R01-telecom
```

Comment:
I am wondering if this could just be an element in the parent schema.  It looks lonely here.